src/hw2/Admin.java

```diff
@@ -0,0 +1,14 @@
+package hw2;
+
+class Admin extends Comics {
```

Comment:
why does your Admin extends Comics? 
does it represent IS-A relationship?

Reply:
I wanted that Admin has access to current book, which Admin need to find for current User. 
Admin a1 = new Admin(batman.currentAuthor(), batman.currentYear(), 1);
So it seems like HAS-A. Need to think about this in HW3.